docs/swagger/spec.yml

```diff
@@ -7235,6 +7235,47 @@ definitions:
       geometry:
         $ref: "#/definitions/Geometry"
 
+  AnnotationFeatureWithOwnerInfo:
+    type: object
+    properties:
+      type:
+        type: string
+      properties:
+        allOf:
```

Comment:
Looks like this is missing `labeledBy` and `verifiedBy` -- do we need that additional information as well for display purposes in the annotation UI?

Reply:
@notthatbreezy good call. I think we need them. They are included in the return. I will add them to the spec as well.